src/AppBundle/Serializer/ContributorNormalizer.php

```diff
@@ -54,10 +59,8 @@ public function normalize($object, $format = null, array $context = []): array
                 'example' => [
                     'matchingUrl' => $exampleNotice->getMatchingContexts()->first()->getExampleUrl(),
                     'noticeId' => $exampleNotice->getId(),
-                    'noticeUrl' => $this->router->generate(
-                      'app_api_getnoticeaction__invoke',
-                      ['id' => $exampleNotice->getId()],
-                      RouterInterface::ABSOLUTE_URL),
+                    'noticeUrl' => $this->noticeUriGenerator->generate($exampleNotice),
+                    'noticeUri' => $this->noticeUriGenerator->generate($exampleNotice),
```

Comment:
Actually. I’d go `URL` :-)  (https://stackoverflow.com/a/1984225/162038)

Reply:
> In computing, a Uniform Resource Locator (URL) is a subset of the Uniform Resource Identifier (URI)
Alright nice catch.

Reply:
Yes. So here `URI` is not wrong per se. But it does not appear to me as being specific enough.

We do have an `URL` here, that enables us to `locate` a notice `resource` in a `unique manner`.

More generally, everytime we do have an `URL`, I don’t see why we’d say `URI` … (even if it is a URI)